### ISSUE TITLE
User Filtering to combat spam

### ISSUE THREAD

**Issue by majestrate** [Nullchan/nullchan#6]
Someone started dumping undesirable content on some of the boards, to combat this it's probably best to add selective end user level filtering based on individual posters.

**Comment by sthetz** [comment]
Good idea and since everyone's able to detect individual posters anyway it's not an infringement of anonymity. Will do.

**Comment by arkre** [comment]
Please take traffic into consideration. I'm new to all this stuff, and don't really understand everything, but my intuition says visual filtering by user indentity will not prevent from space limit ddos by large amount of data.

**Comment by spencerthayer** [comment]
@sthetz If everyone is able to detect individual posters how is that anonymous? Since there is no documentation I haven't been able to glean that from the code as of yet. Thanks!
